[WIP] Add information about alt text (#349)
grafana/writers-toolkit: docs/sources/write/image-guidelines/index.md

1. Run `make docs` on your branch and verify that the image appears.
1. (Optional) Upload a new version and test again.

## Alt text

> Alt text is an HTML attribute that you can use to provide a concise description of an image. The text is used in situations where the image isn't visible, such as people using screen readers, or people who have a low-bandwidth internet connection.

Reviewer: ```suggestion
Alt text is an HTML attribute that you can use to provide a concise description of an image. The text is used in situations where the image isn't visible, such as when people use screen readers or have a low-bandwidth internet connection.
```

In Markdown, the alt text is the text in square brackets when declaring an image:

```markdown
![Alt text](/media/<path-to-image/image-file.png>)
```

> 

Reviewer: I'm thinking we should add an example using a figure shortcode here as well. Speaking of the shortcode, the `alt` parameter is indicated as not required and I'm sure it isn't for the shortcode to work; should we change that or add a note there to clarify what's meant by required in that context? cc @jdbaldry

Reviewer 2: I would like to make it required but it will break the website build if I do since so many invocations do not have that value set.

In my PR, I first emit a log warning which we can use to identify all the misuse. Once that's cleaned up, we can make it required.

Every image that you add to the Grafana documentation should have an alt text.

> A good guidance for writing good alt text from "[HTML: The Living Standard](https://html.spec.whatwg.org/dev/images.html#alt)" is:

Reviewer: ```suggestion
A good piece of guidance for writing good alt text from "[HTML: The Living Standard](https://html.spec.whatwg.org/dev/images.html#alt)" is:
```
